core/src/main/java/nl/nn/adapterframework/webcontrol/action/SendJmsMessage.java

```diff
@@ -53,26 +54,19 @@ public ActionForward executeSub(
 	
 	    DynaActionForm sendJmsMessageForm = getPersistentForm(mapping, form, request);
 	
-	    Cookie[] cookies = request.getCookies();
+	    Cookie cookie = CookieUtil.getCookie(request, AppConstants.getInstance().getProperty("WEB_JMSCOOKIE_NAME"));
+		if (null != cookie) {
+			StringTagger cs = new StringTagger(cookie.getValue());
 	
-	    if (null != cookies) {
-	        for (int i = 0; i < cookies.length; i++) {
-	            Cookie aCookie = cookies[i];
-	
-	            if (aCookie.getName().equals(AppConstants.getInstance().getProperty("WEB_JMSCOOKIE_NAME"))) {
-	                StringTagger cs = new StringTagger(aCookie.getValue());
-	
-	                log.debug("restoring values from cookie: " + cs.toString());
-	                try {
-	                    sendJmsMessageForm.set("jmsRealm", cs.Value("jmsRealm"));
-	                    sendJmsMessageForm.set("destinationName", cs.Value("destinationName"));
-	                    sendJmsMessageForm.set("destinationType", cs.Value("destinationType"));
-	                } catch (Exception e) {
-	                    log.warn("could not restore Cookie value's");
-	                }
-	            }
-	        }
-	    }
+            log.debug("restoring values from cookie: " + cs.toString());
+            try {
+                sendJmsMessageForm.set("jmsRealm", cs.Value("jmsRealm"));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.